src/PaymentMethods/PaymentFacade.php

```diff
@@ -166,8 +166,13 @@ public function paymentMethod(): PaymentMethod
      */
     public function __call(?string $name, array $arguments)
     {
-        if (method_exists($this->paymentMethod, $name))
-        {
+        if (method_exists($this->paymentMethod, $name)) {
```

Comment:
why did you decide to create a new method and not just allow a "version" property that can be added as a request parameter ?